build.gradle

```diff
@@ -1,9 +1,8 @@
 apply plugin: 'java'
-apply plugin: 'maven'
-apply plugin: 'signing'
+apply plugin: 'maven-publish'
 
 group = 'com.tananaev'
-version = '1.2'
+version = '1.3.1'
```

Comment:
There is an already published version `1.3`. Following the convention during other `java-packages` releases, i've bumped the patch version.

Reply:
Config service is using that version:
https://github.com/mindjolt/config-service/blob/542dad999a03b3cb5a4c262969bf62d41d6be3d2/config-api/build.gradle#L88C5-L88C49